RELEASING.md

```diff
@@ -19,9 +24,13 @@
   - make sure the PR title starts with `rel:` to conform with our semantic commits convention
   - add the `no-changelog`, `type: maintenance`, and `version: no bump` labels
 - Once the pull request is approved, squash-merge it
+
+## Git Tags and Releases
 - Checkout `main` and fetch the now-updated `main` branch.
-- Apply a tag for the new version on the merged commit (e.g. `git tag -a v2.3.1 -m "v2.3.1"`)
-- Push the tag upstream (this will kick off the release pipeline in CI) e.g. `git push origin v2.3.1`
+- Apply a tag for the new version on the merged commit (e.g. `git tag -a web-distro-v2.3.1 -m "web-distro-v2.3.1"`). The tag MUST begin with either `web-distro-v` or `react-instrumentation-v` in order to trigger our CI release pipeline
```

Comment:
Does it make sense for our tag names (and build steps) to align more with package names rather than what we call Instead of  `web-distro `honeycomb-opentelemetry-web-vX.Y.Z` or `otel-web-vX.Y.Z` and `honeycomb-opentelemetry-react-user-instrumentation-vX.Y.Z` (or `react-user-instrumentation-vX.Y.Z`)  are more explicit especially as we add more packages.

Reply:
yeah that's fair. on one hand, I'm lazy and don't want to type a lot...but on the other, clarity is nice. I'll make that change.